src/pyuvdata/utils/pol.py

```diff
@@ -812,3 +812,181 @@ def _check_jones_spacing(*, jones_array, strict=True, allow_resort=False):
             "make it impossible to write this data out to calfits."
         )
         tools._strict_raise(err_msg=err_msg, strict=strict)
+
+
+def get_feeds_from_pols(polarization_array):
+    """
+    Return a list of expected feeds based on polarization values.
+
+    Translates values in polarization_array or jones_array into
```

Comment:
sentence fragment.

Reply:
Fixed in 11d373ecdf270b073d714514c43f9a1bd55e21d0.